src/fontstash.h

```diff
@@ -1041,6 +1058,18 @@ static FONSglyph* fons__getGlyph(FONScontext* stash, FONSfont* font, unsigned in
 	// Could not find glyph, create it.
 	scale = fons__tt_getPixelHeightScale(&font->font, size);
 	g = fons__tt_getGlyphIndex(&font->font, codepoint);
+	// Try to find the glyph in fallback fonts.
+	if (g == 0)
+	{
+		for (i = 0; i < font->nfallbacks; ++i)
+		{
+			fallbackGlyph = fons__getGlyph(stash, stash->fonts[font->fallbacks[i]], codepoint, isize, iblur);
+			if (fallbackGlyph != NULL)
```

Comment:
Checking for null only does not sound quite right (but surely should check null too). fons__getGlyph should return null only when the atlas is full. The code is currently made so that even if we hit 0 index (i.e. no glyph found), we still create cache entry for that to avoid potentially costly lookups later. Both freetype and stbtt should return 0 for missing glyphs.